src/resources/Google.ts

```diff
@@ -0,0 +1,8 @@
+export const Google = {
+  analytics: {
+    // DEV, comment out for prod
+    trackingId: 'UA-87988078-4'
```

Comment:
I've figured out a way to do this stuff using ENV variables that we can set on AppCenter depending on the branch we build from. We can implement this pretty quickly if you're not in a rush to release this new version

Reply:
+ No public exposure if these id's :D

Reply:
This is actually the biggest thing I was hoping to talk to you about. These IDs are technically not sensitive (they can be found in the client-facting JS code on the website), but they definitely should be env vars regardless.

I'm inclined to go ahead and release this version anyway, to get testing done by more people sooner.